.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+# Build Docker image using GitHub Actions
+In order to successfully build a Docker image using GitHub Actions, you will need to:
+* [ ] Prepare all necessary files to build the image (see other examples in this repository)
+* [ ] Create, if not already present, `build.env` file (in the same folder as `Dockerfile`) which includes the following tags:
+
+    * `BASE_IMAGE` is the image used to build your Dockerfile, i.e. `BASE_IMAGE=public.ecr.aws/genialis/resolwebio/base:ubuntu-22.04-14112023`.
+    * `REPOSITORY` is the name of the image once built, for example `REPOSITORY=resolwebio/common`.
+    * `TAG` will be the tag of the image, i.e. `TAG=common:0.0.0`.
```

Comment:
I would also add a checkbox to stress the importance of updating the tag on existing images.